### ISSUE TITLE
Option to stop the dropdown repositioning

### ISSUE THREAD

**Issue by garybond** [Bluefieldscom/intl-tel-input#124]
I have an issue where the dropdown is appearing above the target textbox due to its proximity to the bottom of the viewport, but having it render above the textbox is causing it to appear under other controls that necessarily have a higher z-index. I'd like to see an initialise option that turns off the auto repositioning.

**Comment by garybond** [comment]
Sure. I have a questionnaire page containing a number of divs that have a decreasing z-index (from 100 down) to ensure that any popup validation messages inside of a previous question always appear overlapping subsequent questions. Adjusting the z-index of the intl-tel-input select list container has no effect, as the z-index of the question div is lower than its previous sibling.

**Comment by jackocnr** [comment]
Hmm so you're saying it's the question divs that have the decreasing z-index? Sounds like it's not the question itself that needs to be "above" the next question, but just that all of the validation popups need to be above all of the questions?

So instead of
```css
.question1 {z-index: 100;}
.question2 {z-index: 99;}
.question3 {z-index: 98;}
...
```
Couldn't you just have:
```css
.question {z-index: 0;}
.popup {z-index: 1;}
```

Or if this is more complicated than I think, could you provide a link, or a codepen or whatever?

**Comment by garybond** [comment]
I'm hoping that it's more complicated than you think, rather than a trivial oversight on my part, considering the amount of time I invested in finding a fix for the issue in the first place. I'll just throw in here that the validation message popups are ValidatorCalloutExtenders, part of the Microsoft ASP.Net AjaxControlToolkit that spews out loads of inline styles. That's not something that can be changed easily in this legacy product. If the option I suggested is not useful to anyone else and/or not trivial to implement, feel free to close the issue, as I found a css workaround (cough) to force the intl-tel-input select list container to display below the input always. 

.intl-tel-input .country-list { top: initial !important; }
